clients/python/sust/api/climate_explorer/client.py

```diff
@@ -0,0 +1,52 @@
+from sust.api.climate_explorer import clientgen
+from sust.api.climate_explorer.clientgen.api import portfolios_api
+
+
+class Client:
```

Comment:
This class really just wraps up some convenience functions and abstracts the generated client code.

I don't quite know yet how I prefer to expose individual API requests... I'm not in love with creating a function on the client for each one a la "portfolios" and "portfolio_assets" below. One can imagine how this will create a supportability issue.